crates/bevy_reflect/src/list.rs

```diff
@@ -175,6 +208,12 @@ pub fn list_apply<L: List>(a: &mut L, b: &dyn Reflect) {
     }
 }
 
+/// Compares a [`List`] with a [`Reflect`] value.
+///
+/// Returns true _iff_ all of the following are true:
```

Comment:
```suggestion
/// Returns true if and only if all of the following are true:
```
I love `iff`, but it's not clear to readers without a CS background.